structure_tests/structure_test_v1.go

```diff
@@ -15,22 +15,18 @@
 package main
 
 import (
-	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
-	"os/exec"
-	"strings"
-	"syscall"
 	"testing"
 )
 
 type StructureTestv1 struct {
 	GlobalEnvVars      []EnvVar
 	CommandTests       []CommandTestv1
-	FileExistenceTests []FileExistenceTestv1
-	FileContentTests   []FileContentTestv1
-	LicenseTests       []LicenseTestv1
+	FileExistenceTests []FileExistenceTestv0
```

Comment:
Yeah the package versioning will definitely make this a bit easier. You have a PR coming with that soon?

Reply:
Yeah I'm working on a second PR that will abstract out the backend (i.e. Docker) into an interface so that we can support multiple drivers. I'll version the packages there, unless you think I should try and do it here